client/src/locales/en.json

```diff
@@ -79,7 +79,8 @@
   "authorization.sign-in": "Sign in",
   "authorization.sign-up": "Sign up",
   "authorization.sign-up.confirmation-link-sent": "In order to complete sign up process please click the link that we have just sent to {data}.",
-  "authorization.sign-up.button.resend-link": "Resend activation link",
+  "authorization.sign-up.button.resend-link": "Resend link",
```

Comment:
maybe: Resend recovery link

Reply:
It's a common button now.
Displayed when the confrimation and recovery process is running